.gitignore

```diff
@@ -0,0 +1,13 @@
+/.bundle/
+/.yardoc
+/Gemfile.lock
+/_yardoc/
+/coverage/
+/doc/
+/pkg/
+/spec/reports/
+/tmp/
+/vendor/
+/.idea/
+.env
+.DS_Store
```

Comment:
Smirnoff I'm happy for you to remove any IDE/OS related files from this PR and flow the team on what file/commands they should run

I agree, this file should be related to this project, e.g. coverage, Gemfile.lock, etc.